packages/sources/css/src/components/radio-button/src/index.css

```diff
@@ -0,0 +1,86 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+.vtmn-radio-button [type='radio'] {
+  left: 0;
+  opacity: 0.01;
+}
+
+.vtmn-radio-button [type='radio'] + label {
+  position: relative;
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  line-height: var(--vtmn-typo_text-2-line-height);
+  cursor: pointer;
+  user-select: none;
+  left: rem(-24px);
+  padding: 6px 8px 8px 28px;
+}
+
+.vtmn-radio-button [type='radio'] + label::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  top: 6px;
+  width: 20px;
+  height: 20px;
+  border: 2px solid;
+  border-radius: 100%;
+  transition: all 0.275s;
+}
+
+.vtmn-radio-button [type='radio']:not(:checked) + label::before {
+  border-color: var(--vtmn-color_grey-light-1);
+}
+
+.vtmn-radio-button
+  [type='radio']:not(:checked):not(:disabled):hover
+  + label::before {
+  box-shadow: 0 0 0 7px rgba(0, 16, 24, 0.05);
```

Comment:
We can use the color-mod method to keep our vitamin color variables :
box-shadow: 0 0 0 rem(7px) color-mod(var(--vtmn-color_grey-dark-1) alpha(5%));